### PR TITLE
Feat/history

### PR DESCRIPTION
**Pull request by matiu**
- updates function call signatures
- add pagination to wallet-history 
- add support for encrypted json imports (from Copay)
- add --output (JSON output) to wallet-history

### CODE REVIEW

**Comment by matiu** [comment]
updated.

Related to https://github.com/bitpay/bitcore-wallet-client/pull/257

File: bin/wallet-history [hunk 1]
```diff
@@ -4,39 +4,96 @@ var _ = require('lodash');
...
-        break;
-        case 'moved':
-          direction = '==';
-        break;
-        case 'sent':
-          direction = '=>';
-        break;
-      }
+var skip = +program.skip || 0,
+  page = +program.page || 55,
```
Comment (isocolsky) [inline bin/wallet-history]: 55?

File: bin/wallet-import [hunk 2]
```diff
@@ -2,14 +2,16 @@
 
 var program = require('commander');
 var fs = require('fs');
+var sjcl = require('sjcl');
 var utils = require('./cli-utils');
 program = utils.configureCommander(program);
 
 program
   .option('-t, --testnet', 'testnet network')
-  .option('-f, --file', 'import from file')
```
Comment (isocolsky) [inline bin/wallet-import]: Nice catch.

File: bin/wallet-history (hunk 1 above)
Comment (matiu) [inline bin/wallet-history]: 55 is the 10th Fibonacci number and a triangular number (the sum of the consecutive numbers 1 to 10). It is the largest Fibonacci number to also be a triangular number.

Comment (matiu) [inline bin/wallet-history]: And also matches current Copay's page size

Comment (isocolsky) [inline bin/wallet-history]: I wouldn't mind if 55 -being such a beautiful number- matched Copay's page size. What bothers me is that it matches Copay's default page... that is just unacceptable...

**Comment by matiu** [comment]
updated.

**Pull request closed without merging**